src/components/Calculator.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './Calculator.css';
+
+// Button component
+function Button({ value, className = 'button', onClick }) {
+  const handleClick = () => {
+    onClick(value);
+  };
+
+  return (
+    <button type="button" className={className} onClick={handleClick}>
+      {value}
+    </button>
+  );
+}
```

Comment:
[OPTIONAL] I think you should create a separate file for the Button component. This keeps your code simple and helps with separation of concerns